tests/termsAndConditions.js

```diff
@@ -0,0 +1,61 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../src/index';
+import statusCode from '../src/helpers/constants/status.codes';
+
+const { expect } = chai;
+chai.use(chaiHttp);
+let userToken = '';
+
+describe('test the rating controller', () => {
+  it('should login a user and get a token to use against on rating article', (done) => {
```

Comment:
is it article's rating or terms and conditions?

Reply:
sorry I don't know how I wrote this .it is a mistake  going to change it right now

Reply:
done

Reply:
@murediane I can still see `rating article` here